configs/openSUSE/permissions-whitelist.toml

```diff
@@ -39,13 +39,21 @@ package = "sendmail"
 type = "permissions"
 note = """Be careful about calculating digest sums here, the sendmail package
       replaces file contents during RPM build time with architecture dependent
-      values"""
+      values.
+      The duplicate entries are due to the UsrMerge. We can remove the /etc
+      entries once the dust has settled. See bsc#1219339"""
 [[FileDigestGroup.digests]]
 path = "/etc/permissions.d/sendmail"
 hash = "e09ca5efebd0b3c123afc2364f9745f4d85c4327fa83f709bccbaa64da764486"
 [[FileDigestGroup.digests]]
 path = "/etc/permissions.d/sendmail.paranoid"
 hash = "2d5c56cdfb00ec169c182de791cf2934331159842f1849c5f2d7059f0086bd2c"
+[[FileDigestGroup.digests]]
+path = "/usr/share/permissions/permissions.d/sendmail"
+hash = "e09ca5efebd0b3c123afc2364f9745f4d85c4327fa83f709bccbaa64da764486"
+[[FileDigestGroup.digests]]
+path = "/usr/share/permissions/permissions.d/sendmail.paranoid"
+hash = "2d5c56cdfb00ec169c182de791cf2934331159842f1849c5f2d7059f0086bd2c"
```

Comment:
An alternative might be to used file globbing:

`path = "glob:{/etc,/usr/share/permissions}/permissions.d/sendmail`. Don't know if it's more intuitive, it's less redundant in any case.

Reply:
Reasonable; updated.